examples/solid-client-access-grants-demo/README.md

```diff
@@ -0,0 +1,49 @@
+# Solid Access Requests and Access Grants Demo
+
+## Run the demo app
+
+- Run `npm ci` and `npm run build` in the current directory to build the demo app.
+- Create a `.env.local` file based on `.env.local.example`.
+- Fill in the required settings following the instructions in `.env.local.example`.
+- Start the app `npm start`.
+
+## Run the demo app using the local library
+
+- Instead of `npm run build`, use `npm run build:local` to build the demo app using
+  the local `solid-client-access-grant-js` code (instead of the installed dependency
+  `@inrupt/solid-client-access-grants`).
+- All other steps remain the same as in "[Run the app](#run-the-app)"
+
+## Access flow
+
+Let the application be ACME.
+Let the user be Alice.
+Let the access management app be PodBrowser).
```

Comment:
remove extra )

Reply:
Addressed in https://github.com/inrupt/solid-client-access-grants-js/pull/252/commits/393f0b4ee8cc698a7eac3f2e5cd850b28d784840